tests/test.sh

```diff
@@ -21,7 +28,7 @@ function cleanup {
 }
 
 # Calls cleanup when the script exits
-trap cleanup EXIT
+#trap cleanup EXIT
```

Comment:
I'd suggest wrapping this in a conditional so that the trap is activated only if there's no custom directory specified